.github/workflows/tests.yml

```diff
@@ -169,6 +178,24 @@ jobs:
           # Fix working directory permissions
           docker exec --user root linuxbrew chmod 777 /tmp/bottles
 
+      - name: Run Linux Wheezy Docker container
```

Comment:
This duplication here is a bit nasty. I don't mind it shipping as-is for now but would really love to get all this logic into Homebrew/actions.

Reply:
Duplication is no more. But agree on the action side.

Reply:
Resolved for now, to not clutter the view.